extension/controller/experiments/LanguageDetector/api.js

```diff
@@ -27,7 +27,16 @@ this.experiment_languageDetector = class extends ExtensionAPI {
         languageDetector: {
           detect: async function detect(str) {
             try {
-              return await LanguageDetector.detectLanguage(str);
+              let lang = await LanguageDetector.detectLanguage(str);
+
+              /*
+               * language detector returns "no" for both Norwegian Nynorsk ("nn") and Norwegian Bokmål ("nb")
+               * let's default to "nb", since we have a better model and
+               * Bokmål is more popular in Norway, about 85-90 % of writing is done in Bokmål.
```

Comment:
> Bokmål is more popular in Norway, about 85-90 % of writing is done in Bokmål.

@eu9ene Do you have any evidence to back this up?

Reply:
CLD2 ships two models, a small one and a full one.  If you're getting `no` it means you're using the small model.  The full model gives `nn` and `nb`.  Related to #30.  

See https://github.com/bitextor/warc2text/commit/e2543e4d2086a29c914c564aae885e7dcb888dbf for an example of changing which model is used.

Reply:
Maybe @lars-t-hansen can provide us some tips here?

Reply:
> > > Bokmål is more popular in Norway, about 85-90 % of writing is done in Bokmål.
> > 
> > 
> > @eu9ene Do you have any evidence to back this up?
> > Plus, we should create an issue for ourselves to not forget about updating this code whenever we integrate a better Language detector.
> 
> I don't have "evidence" but some random internet articles say that Bokmal is more popular like this one https://norwegianacademy.com/nynorsk-or-bokmal/.

Maybe @lars-t-hansen can provide us some tips here?

Reply:
Bokmål is the dominant idiom.  Norwegians will tell you that including those in the know like the National Library.  

Regarding the web, just look at the file sizes of monolingual dumps here:
https://data.statmt.org/heafield/classified-fasttext/wide00016/nno/
https://data.statmt.org/heafield/classified-fasttext/wide00016/nor/
Much to my chagrin, the fasttext classifier used for that does `nor` when it should emit `nob`.  I have confirmed with the author of the LID used that it `nor` refers to Bokmål.  

That said, you have a classifier that tells you the difference...

Reply:
Bokmål is indeed the dominant written idiom (85-90% sounds about right, possibly even low) so for sheer applicability it would be useful to go after that first, but it would be politically hazardous to recognize bokmål and call it simply "Norwegian".

Reply:
> Bokmål is indeed the dominant written idiom (85-90% sounds about right, possibly even low) so for sheer applicability it would be useful to go after that first, but it would be politically hazardous to recognize bokmål and call it simply "Norwegian".

Thank you for your input! It is called properly in the infobar UI, the only issue is that our language detector cannot distinguish between two idioms, so we default to Bokmål for now. I hope we will fix it in #30.


<img width="551" alt="Screen Shot 2022-02-15 at 10 07 17 AM" src="https://user-images.githubusercontent.com/2486505/154123222-be8e5672-929c-4f0f-a2fd-3025b359bd2b.png">